### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by ProPontifex**
See title

### CODE REVIEW

**Comment by ProPontifex** [comment]
Be wary: I REALLY don't know what cc4618c is about as it just appeared. If erroneous, please delete.

**Comment by ProPontifex** [comment]
No, I didn't: if you can reverse it, please do.

**Review by BellezaEmporium — COMMENTED** [review]
Infriging commit is the merge branch.

cc4618c4003fe715d232d65dca2149dda91a3332

**Comment by BellezaEmporium** [comment]
I tried reversing it, but weirdly enough it... just re-added the changes Arhey made to the blocklist, ignoring the channels part.

bb8ec270ca4df2e92613747d3523d923ca0cdfed contains the channels.csv changes that were removed. I'll try fixing that from the conflicts.

**Comment by ProPontifex** [comment]
Thank you, Belleza.

**Comment by BellezaEmporium** [comment]
My PR view from it seems... clean. Wonder why it says there's 200 more deleted lines.

**Comment by BellezaEmporium** [comment]
If i've done this properly, it should be good.

**Comment by ProPontifex** [comment]
All is well: merci beaucoup!

**Comment by ProPontifex** [comment]
Yes: it needs to be readded.

File: data/channels.csv
Comment (freearhey) [inline data/channels.csv:21550]: Why replace `ntv.at` with `ntv.de`?

File: data/channels.csv
Comment (freearhey) [inline data/channels.csv:21572]: Why delete `ntvHD.at`?

File: data/channels.csv
Comment (freearhey) [inline data/channels.csv:21529]: Needs to be added back in before approval.

**Review by freearhey — CHANGES_REQUESTED** [review]

File: data/channels.csv
Comment (BellezaEmporium) [reply data/channels.csv:21550]: The replacement was needed due to the fact that the channel was never Austrian to begin with, and it always was German. That's what AP told me.
image

Though https://oesterreichprogramm.at/ntv-austria/ seems to indicate there's a branch in Austria... since pretty recently (on satellite since August of this year).

File: data/channels.csv
Comment (ProPontifex) [reply data/channels.csv:21572]: Please read the Wikipedia article for clarification; also, please approve.

File: data/channels.csv
Comment (ProPontifex) [reply data/channels.csv:21550]: I stand corrected: add the Austrian entry back in.

**Comment by ProPontifex** [comment]
ntv.at has been submitted to the database

**Review by freearhey — CHANGES_REQUESTED** [review]
The `ntv.at` is already in the database, you just need to update your pull request and put the channel entry back in the file.

**Review by BellezaEmporium — APPROVED** [review]
Now, we're good.

**Review by freearhey — APPROVED** [review]

**Pull request merged**
